tpm2/constants.go

```diff
@@ -29,6 +29,11 @@ func init() {
 // Algorithm represents a TPM_ALG_ID value.
 type Algorithm uint16
 
+// IsNull returns true if a is AlgNull or zero (unset).
+func (a Algorithm) IsNull() bool {
+	return a == AlgNull || a == 0
```

Comment:
For clarity in the code, how about naming the 0 case AlgUnset or AlgUnknown?

Reply:
Done